src/utils/metamaskHelpers.ts

```diff
@@ -1,4 +1,6 @@
-const addNetwork = (params: any) => {
+export const metamaskDefaultChains = [1, 3, 4, 5, 42];
+
+export const addNetwork = (params: any) => {
   const { ethereum } = window as any;
```

Comment:
You could export `ethereum` the same way as `metamaskDefaultChains` so you don't have to redeclare it in every method.